common/src/main/java/com/example/demo/utils/ShopConstant.java

```diff
@@ -0,0 +1,11 @@
+package com.example.demo.utils;
+
+public class ShopConstant {
+
+    private ShopConstant() {
+
+    }
+
+    public static final int STOCK_MIN_CAPACITE = 0;
```

Comment:
If you only need to store constants, you can just create an interface (no constructor) and by default, all the values are public static final so you need to specify them as : 
```JAVA
int STOCK_MIN_CAPACTITY = 0;
```

Reply:
very good idea on the use of interface. I will implement this 😊.